sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/SqlBasedBenchmark.scala

```diff
@@ -34,6 +35,7 @@ trait SqlBasedBenchmark extends BenchmarkBase with SQLHelper {
     SparkSession.builder()
       .master("local[1]")
       .appName(this.getClass.getCanonicalName)
+      .config(EXECUTOR_EXTRA_JAVA_OPTIONS, "-XX:+UseParallelGC -XX:-UseDynamicNumberOfGCThreads")
```

Comment:
It seeems `EXECUTOR_EXTRA_JAVA_OPTIONS` not works for local mode. We need to use `DRIVER_EXTRA_JAVA_OPTIONS`:
![image](https://user-images.githubusercontent.com/5399861/65853774-2d373f00-e38d-11e9-9ed2-e09bea8aee9d.png)
![image](https://user-images.githubusercontent.com/5399861/65853856-78e9e880-e38d-11e9-90cb-2f8102d9566b.png)

Reply:
Thanks!

Reply:
? Did you run that without `sbt`? I'm using `sbt` and the following is the result. We already have the proper setting in `SparkBuild.scala`.

<img width="690" alt="sbt_result" src="https://user-images.githubusercontent.com/9700541/65896522-38c14080-e362-11e9-9b57-f233a9887c8c.png">

Reply:
I see. Is `.config(EXECUTOR_EXTRA_JAVA_OPTIONS, "-XX:+UseParallelGC -XX:-UseDynamicNumberOfGCThreads")` an invalid conf?

Reply:
@wangyum . In `sbt` testing environment, everything is already applied. Please let me know again if your comment is relevant to this PR.

Reply:
Could we remove this conf? Because it is useless.

Reply:
Got it. Sure. It looks not working as intended. I'll remove that and regenerate. Thanks, @wangyum .